index.js

```diff
@@ -444,6 +446,7 @@ AFRAME.registerComponent('lounge', {
     depth: {type: 'number', default: 7},
     floorColor: {type: 'color', default: ''},
     floorTexture: {type: 'asset', default: ''},
+    floorRepeatTexture: {type: 'vec2', default: {x: 1, y: 1}},
```

Comment:
Since for now we only have the texture that can be repeated, and I don't envision anything else that could be repeated, would you mind using "floorRepeat" instead of "floorRepeatTexture"? This way it would also be more consistent with the name used for the `material` property (`repeat`).